modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/ModuleProcess.cs

```diff
@@ -166,30 +171,32 @@ public class ModuleProcess : IProcessControl {
         /// <summary>
         /// Set stack trace mask based on log level.
         /// </summary>
-        public void SetStackTraceMask() {
+        public void SetStackTraceMask(Extensions.Logging.ILogger logger) {
+            OpcStackTraceMask = Utils.TraceMasks.Error | Utils.TraceMasks.Security;
             switch (LogControl.Level.MinimumLevel) {
                 case LogEventLevel.Fatal:
-                    OpcStackTraceMask = 0;
+                    OpcStackTraceMask |= 0;
                     break;
                 case LogEventLevel.Error:
-                    OpcStackTraceMask = Utils.TraceMasks.Error;
+                    OpcStackTraceMask |= Utils.TraceMasks.StackTrace;
                     break;
                 case LogEventLevel.Warning:
-                    OpcStackTraceMask = 0;
+                    OpcStackTraceMask |= Utils.TraceMasks.StackTrace;
                     break;
                 case LogEventLevel.Information:
-                    OpcStackTraceMask = 0;
+                    OpcStackTraceMask |= Utils.TraceMasks.StartStop | Utils.TraceMasks.StackTrace | Utils.TraceMasks.Information;
```

Comment:
Is there a concern that in this case the stack will be too noisy ? I am asking because previously this mask was applied in case of `Debug` log level and now it is `Information`.

Reply:
OK, sounds good.